src/backend/src/prisma/schema.prisma

```diff
@@ -408,4 +408,5 @@ model User_Secure_Settings {
   city                 String
   state                String
   zipcode              String
+  phoneNumber          String @unique
```

Comment:
add this to the existing finance migration instead of making a whole new migration